clients/unshaded/src/main/java/tachyon/hadoop/AbstractTFS.java

```diff
@@ -318,11 +319,66 @@ public FileStatus getFileStatus(Path path) throws IOException {
 
     FileStatus ret = new FileStatus(fileStatus.getLength(), fileStatus.isIsFolder(),
         BLOCK_REPLICATION_CONSTANT, fileStatus.getBlockSizeBytes(), fileStatus.getCreationTimeMs(),
-        fileStatus.getCreationTimeMs(), null, null, null, new Path(mTachyonHeader + tPath));
+            fileStatus.getCreationTimeMs(), new FsPermission((short) fileStatus.getPermission()),
+            fileStatus.getUserName(), fileStatus.getGroupName(), new Path(mTachyonHeader + tPath));
     return ret;
   }
 
   /**
+   * Changes owner or group of a path (i.e. a file or a directory). If username is null, the
+   * original username remains unchanged. Same as groupname.
+   *
+   * @param path path to set owner or group
+   * @param username username to be set
+   * @param groupname groupname to be set
+   * @throws IOException if the path failed to be changes owner or group
+   */
+  @Override
+  public void setOwner(Path path, final String username, final String groupname)
+      throws IOException {
+    TachyonURI tPath = new TachyonURI(Utils.getPathWithoutScheme(path));
+    Path hdfsPath = Utils.getHDFSPath(tPath, mUnderFSAddress);
+    LOG.info("setOwner({},{},{}) HDFS Path: {} TPath: {}{}", path, username, groupname, hdfsPath,
```

Comment:
Thank you for your comments, I found other methods like `getFileStatus` also use `LOG.info`. How about keep the same log level with other commands?